tests/php/bootstrap.php

```diff
@@ -1,47 +1,44 @@
 <?php
 
+define( 'TEST_DOKAN_PLUGIN_DIR', dirname( __DIR__, 2 ) );
+define( 'TEST_WC_DIR', dirname( TEST_DOKAN_PLUGIN_DIR, 1 ) . '/woocommerce' );
+
 // Composer autoloader must be loaded before WP_PHPUNIT__DIR will be available
-require_once dirname( __FILE__, 2 ) . '/vendor/autoload.php';
+require_once TEST_DOKAN_PLUGIN_DIR . '/vendor/autoload.php';
+
 $_tests_dir = getenv( 'WP_TESTS_DIR' ) ? getenv( 'WP_TESTS_DIR' ) : getenv( 'WP_PHPUNIT__DIR' );
 
-if ( !$_tests_dir ) {
+if ( ! $_tests_dir ) {
     $_tests_dir = rtrim( sys_get_temp_dir(), '/\\' ) . '/wordpress-tests-lib';
 }
 
-if ( !file_exists( $_tests_dir . '/includes/functions.php' ) ) {
+if ( ! file_exists( $_tests_dir . '/includes/functions.php' ) ) {
     echo "Could not find $_tests_dir/includes/functions.php, have you run bin/install-wp-tests.sh ?" . PHP_EOL; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
     exit( 1 );
 }
 
 // Give access to tests_add_filter() function.
 require_once $_tests_dir . '/includes/functions.php';
 
-define( 'TEST_WC_DIR', __DIR__ . '/../../woocommerce' );
-define( 'TEST_DOKAN_DIR', dirname( __DIR__ ) );
-
 function _manually_load_plugin() {
     define( 'WC_TAX_ROUNDING_MODE', 'auto' );
     define( 'WC_USE_TRANSACTIONS', false );
 
     require TEST_WC_DIR . '/woocommerce.php';
-    require TEST_DOKAN_DIR . '/dokan.php';
+    require TEST_DOKAN_PLUGIN_DIR . '/dokan.php';
 }
 
 tests_add_filter( 'muplugins_loaded', '_manually_load_plugin' );
 
 function install_wc() {
-    // clean existing install first
-
     define( 'WP_UNINSTALL_PLUGIN', true );
     define( 'WC_REMOVE_ALL_DATA', true );
 
-    include TEST_WC_DIR . '/uninstall.php';
+    // include TEST_WC_DIR . '/uninstall.php';
 
-    WC_Install::install();
+    // WC_Install::install();
```

Comment:
We must ensure the Fresh setup for WooCommerce for every run. We need to run the `uninstall.php` file and then run `WC_Install::install();`

Reply:
> The WordPress Test Library automatically rolls back database changes and other items like hooks and option values between tests.

**Isolate tests from each other**:  https://developer.woocommerce.com/testing-extensions-and-maintaining-quality-code/best-practices-for-unit-testing/